test/integration/patients/worklist/reduced-schedule.js

```diff
@@ -254,11 +254,16 @@ context('reduced schedule page', function() {
       .routeFlow()
       .routeFlowActions()
       .routePatientByFlow()
+      .routeAction()
+      .routeActionComments()
+      .routeActionActivity()
```

Comment:
Doesn't seem like I should have to add these, but it fails without it when using the `wait()`s I added below.